src/studio/src/designer/backend.Tests/Controllers/RepositorySettingsControllerTests.cs

```diff
@@ -28,6 +29,10 @@ public class RepositorySettingsControllerTests : IClassFixture<WebApplicationFac
         public RepositorySettingsControllerTests(WebApplicationFactory<RepositorySettingsController> factory)
         {
             _factory = factory;
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
+            {
+                Environment.SetEnvironmentVariable("ALTINN_KEYS_DIRECTORY", Path.Combine(System.Environment.GetFolderPath(System.Environment.SpecialFolder.ApplicationData), "ASP.NET", "DataProtection-Keys"));
```

Comment:
This block is repeated at least 3 times. Maybe extract it to a static function?

Reply:
The reason I chose to not extract it was that the dirty hack should be visible, but finding all the places it is used is easier if extract. Extracted it to a static function